docs/api/1.x/timestamps.rst

```diff
@@ -81,28 +81,44 @@ the ``If-None-Match: *`` request header.
-| Id unknown                  | Create      | Create       | No effect     | No effect     |
-+-----------------------------+-------------+--------------+---------------+---------------+
++-----------------------------+-------------+-------------+-------------+-------------+-------------+
+|                             | GET         | POST        | PUT         | PATCH       | DELETE      |
++=============================+=============+=============+=============+=============+=============+
+|| **If-Match: "timestamp"**                                                                        |
++-----------------------------+-------------+-------------+-------------+-------------+-------------+
+| Not changed                 | Fetch       | Fetch       | Overwrite   | Modify      | Delete      |
++-----------------------------+-------------+-------------+-------------+-------------+-------------+
+| Changed meanwhile           | ``HTTP 412``| ``HTTP 412``| ``HTTP 412``| ``HTTP 412``| ``HTTP 412``|
++-----------------------------+-------------+-------------+-------------+-------------+-------------+
+|| **If-Match: ***                                                                                  |
++-----------------------------+-------------+-------------+-------------+-------------+-------------+
+| Id exists                   | Fetch       | Fetch       | Overwrite   | Modify      | Delete      |
++-----------------------------+-------------+-------------+-------------+-------------+-------------+
+| Id unknown                  | ``HTTP 412``| ``HTTP 412``| ``HTTP 412``| ``HTTP 412``| ``HTTP 412``|
++-----------------------------+-------------+-------------+-------------+-------------+-------------+
+|| **If-None-Match: ***                                                                             |
++-----------------------------+-------------+-------------+-------------+-------------+-------------+
+| Id exists                   | ``HTTP 412``| ``HTTP 412``| ``HTTP 412``| ``HTTP 412``| ``HTTP 412``|
++-----------------------------+-------------+-------------+-------------+-------------+-------------+
+| Id unknown                  | No effect   | Create      | Create      | No effect   | No effect   |
++-----------------------------+-------------+-------------+-------------+-------------+-------------+
+
 
 When the client receives a |status-412|, it can then choose to:
 
 * overwrite by repeating the request without concurrency control;
 * reconcile the resource by fetching, merging and repeating the request.
 
 
+.. note::
+
+    ``If-Match`` and ``If-None-Match`` headers on Kinto work slightly different from what
+    `their RFC specification expects <https://tools.ietf.org/html/rfc7232#section-3>`_.
+    Instead of comparing if the current
+    ``ETag`` is exactly the same as the value provided on the header,
+    the current implementation just checks if the ``ETag`` is newer or older
```

Comment:
I think there is something wrong with this sentence, since we contrast *exactly the same* with *newer or older*, and that sounds weird. In other words, how *«newer or older»* is not equivalent to *«not exactly the same»*?

By the way, personnally, I would change the behaviour to match the spec. I don't see any implication in the synchronization process. The clients are not supposed to have timestamps that are superior to the server ones, so I wouldn't even consider this a breaking change.